src/main/java/com/google/cloud/spanner/pgadapter/statements/JdbcMetadataStatementHelper.java

```diff
@@ -49,8 +50,13 @@ static String replaceJdbcMetadataStatement(String sql) {
     if (sql.startsWith(PgJdbcCatalog.PG_JDBC_GET_SCHEMAS_PREFIX)) {
       return replaceGetSchemasQuery(sql);
     }
-    if (sql.equals(PgJdbcCatalog.PG_JDBC_GET_EDB_REDWOOD_DATE_QUERY)) {
```

Comment:
This statement was also for Liquibase, but is no longer needed, as we have a generic solution for `pg_setting`. `pg_setting` is automatically added to any query that references it as a common table expression.